lib/compiler-singlepass/src/compiler.rs

```diff
@@ -228,7 +207,7 @@ impl Compiler for SinglepassCompiler {
             .values()
             .collect::<Vec<_>>()
             .into_par_iter_if_rayon()
-            .map(|func_type| gen_std_trampoline(func_type, target, calling_convention))
+            .map(|func_type| gen_std_trampoline(func_type, target, calling_convention).unwrap())
```

Comment:
This will now panic if the chipset is not supported (for example, when cross compiling). Do we really want this?

Reply:
it was panicking before too.

Also, note that this is only on building the tranpoline, that will not panic if chipset is unsupported, as it's more permissive.

(Anyway, there is no easy way to get the Error to interupt the rayon iterator)